2025-08-29 13:56:40 INFO root Start primary API version 1.8.21
2025-08-29 13:56:40 INFO root loading script: main
2025-08-29 13:56:41 INFO root successfully loaded script: main
2025-08-29 13:57:10 ERROR root Receive failed, from client
2025-08-29 13:57:30 WARNING root Bezel button info not found for: device 4-0
2025-08-29 13:57:39 ERROR root Failed to run wait callback function: Wait.w
Traceback (most recent call last):
  File "/remotelib_qxi/__init__.py", line 280, in w
    self.receive_message(data,ipaddress)
  File "/remotelib_qxi/__init__.py", line 344, in receive_message
    WrapperBasics.check_create_objects_queue()
  File "/remotelib_qxi/__init__.py", line 268, in check_create_objects_queue
    dev = WrapperBasics.constructors[data['device type']](instance,alias,data)
  File "/remotelib_qxi/device/UIDevice.py", line 66, in __init__
    self.check_where_used()
  File "/remotelib_qxi/device/UIDevice.py", line 84, in check_where_used
    self.where_used_items[type][id] = self.item_constructors[type](self.WrapperBasics,data)
TypeError: ObjectWrapper.__init__() missing 1 required positional argument: 'data'
2025-08-29 13:57:40 INFO root Device 4-0 VTLP sync handling began
2025-08-29 13:57:40 INFO root Device 4-0 VTLP sync handling ended
2025-08-29 13:57:53 ERROR root Debug Server restarted:Listening
2025-08-29 13:58:14 INFO root Device 4-0 VTLP sync handling began
2025-08-29 13:58:14 INFO root Device 4-0 VTLP sync handling ended
2025-08-29 13:58:14 INFO root Device 4-0 VTLP sync handling began
2025-08-29 13:58:14 INFO root Device 4-0 VTLP sync handling ended
2025-08-29 13:58:23 ERROR root Debug Server restarted:Listening
